packages/eslint-config/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@xt-pagesource/eslint-config",
+  "version": "1.0.0",
+  "description": "XT PageSource (ESLint) configuration",
+  "keywords": [
+    "eslint",
+    "prettier",
+    "rules",
+    "lint",
+    "javascript"
+  ],
+  "main": "index.js",
+  "repository": "https://github.com/pagesource/quality-control.git",
+  "author": "Sapient XT",
+  "license": "MIT",
+  "files": [
+    "index.js"
+  ],
+  "dependencies": {
+    "eslint": "^6.8.0",
+    "eslint-config-airbnb": "^18.1.0",
```

Comment:
We also need to find a way to add all dependencies automatically to the consumer of this package. 

Eg: I install this package in my app, the needed dependencies and scripts will be automatically added to my package json.

Lets talk about it if you need any help.